src/SFA.DAS.QnA.Api/Controllers/SectionsController.cs

```diff
@@ -57,5 +57,22 @@ public async Task<ActionResult<Section>> GetSection(Guid applicationId, Guid sec
 
             return sectionsResponse.Value;
         }
+
+        /// <summary>
+        ///     Returns the requested Section
+        /// </summary>
+        /// <returns>The requested Section</returns>
+        /// <response code="200">Returns a Section</response>
+        /// <response code="404">If the ApplicationId or SectionNo are invalid</response>
+        [HttpGet("{applicationId}/sections/{sectionNo:int}")]
```

Comment:
Think we should include the _sequenceNo_ parameter as an extra check (because _sectionNo_ is non-unique)
For example: Section 2 could end up in both Sequence 1 and Sequence 2

Reply:
Fixed it myself 😀